content/advent-2017/you-cant-do-that-in-go.md

```diff
@@ -0,0 +1,215 @@
++++
+author = ["David Crawshaw"]
+title = "You Can't Do That In Go"
+date = 2017-12-26T00:00:01Z
+series = ["Advent 2017"]
++++
+
+[Neugram](https://neugram.io) is a scripting language that sticks very
+close to Go.
+Go statements are Neugram statements, you can import Go packages,
+scripts can be compiled to Go programs, and types look just like the
+equivalent Go types at run time (which means packages built on
+reflection, like `fmt`, work as expected).
+These requirements put a lot of restrictions on the design of Neugram.
+This post is about one such restriction on methods that I did not
+discover until I tried to use it without thinking.
+
+# Background: Go without declarations
+
+When designing a language for use in a REPL you want the user to be
+able to dive right in and have their code executed as quickly as
+is reasonable.
+So the common choice (Perl, Python, etc) make is to use statements
```

Comment:
Perhaps loose the parens around Perl... etc?
It makes it harder to the sentence otherwise (at least to me, non native English speaker)

Reply:
I tried to clean up the intro. PTAL.

Reply:
LGTM